helm/hpcc/values.yaml

```diff
@@ -68,10 +68,11 @@ global:
 
   cost:
     currencyCode: USD
-    perCpu: 0.126
-    storageAtRest: 0.0135
-    storageReads: 0.0485
-    storageWrites: 0.0038
+    # The following are example pricing based on standard Azure pricing and should be updated to reflect actual rates
+    perCpu: 0.0565        # D64ds_v4 compute node ($2,639.68/month for 64 vCPU)
+    storageAtRest: 0.0208 # Blob storage pricing (East US/Flag NS/LRS redundancy/Hot)
+    storageReads: 0.004   # Blob storage pricing (East US/Flag NS/LRS redundancy/Hot)
+    storageWrites: 0.005  # Blob storage pricing (East US/Flag NS/LRS redundancy/Hot)
```

Comment:
These write (and read) numbers do not look right and don't match the values I can find - please double check them.  The write cost is almost always 10x the read cost...